api/schema/wallet.yml

```diff
@@ -0,0 +1,151 @@
+openapi: 3.0.3
+info:
+  title: Wallet - OpenAPI 3.0
+  description: |-
+    Wallet 
+  termsOfService: http://pagopa.it/terms/
+  contact:
+    email: apiteam@swagger.io
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+externalDocs:
+  description: Find out more about Swagger
+  url: http://swagger.io
+servers:
+  - url: https://wallet.pagopa.it/api/v1
+tags:
+  - name: Wallet
+    description: Intercat with wallet
+paths:
+  /wallets:
+    post:
+      tags:
+        - wallets
+      summary: Add a new wallet
+      description: Add a new wallet
+      operationId: addWallet
+      parameters:
+        - in: header
+          name: x-user-id
+          schema:
+            type: string
+          required: true
+      requestBody:
+        description: Create a new wallet
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/WalletPost'
+        required: true
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/WalletPostResponse'        
+        '405':
+          description: Invalid input
+  /wallets/{walletId}:
+    put:
+      tags:
+        - wallets
+      summary: Update an existing wallet
+      description: Update an existing wallet by Id
+      operationId: updateWallet
+      parameters:
+        - name: walletId
+          in: path
+          description: ID of wallet to return
+          required: true
+          schema:
+            type: integer
+            format: int64
+      requestBody:
+        description: Update an existent wallet
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Wallet'
+        required: true
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Wallet'
+        '400':
+          description: Invalid ID supplied
+        '404':
+          description: wallet not found
+        '405':
+          description: Validation exception
+    get:
+      tags:
+        - wallets
+      summary: Find wallet by ID
+      description: Returns a single pet
+      operationId: getWalletById
+      parameters:
+        - name: walletId
+          in: path
+          description: ID of wallet to return
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Wallet'    
```

Comment:
The GetWalletById Response should have a different model than UpdateWallet requestBody. Maybe the GetWalletById should have the same response body than UpdateWallet